dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -600,4 +600,11 @@ ssl.dynamicSelection.conflict.CWPKI0815W.useraction=Review the conflicting Outbo
 ssl.defaultOutbound.conflict.CWPKI0816W=CWPKI0816W: The outboundConnection element with an asterisk (*) as the host and port is set to the [{0}] SSL configuration. This setting is in conflict with the outboundSSLRef attribute setting. The {1} SSL configuration that is specified by the outboundSSLRef attribute is used.
 ssl.defaultOutbound.conflict.CWPKI0816W.explanation=The outboundConnection element with an asterisk (*) as the host and port is in conflict with the outboundSSLRef attribute that is configured. The server uses the SSL configuration that is specified by the outboundSSLRef attribute.
 ssl.defaultOutbound.conflict.CWPKI0816W.useraction=Review the conflicting configuration and determine which configuration to use as the default SSL configuration. Remove the configuration that you do not need.
-
+#--------------------------------------------------------------------------------------------------
+ssl.defaultKeyStore.expected.CWPKI0817W=CWPKI0817W: The default SSL configuration expects a <keyStore> element with an ID of {0} and a password. The {0} <keyStore> element is missing or the password is not specified. Either define the missing element: <keyStore id="{0}" password="..." /> or change the default SSL configuration to use an existing keystore: <ssl id="defaultSSLConfig" keyStoreRef="newKeyStore" />.
+ssl.defaultKeyStore.expected.CWPKI0817W.explanation=The default SSL configuration expects the default keyStore element which does not exist.
+ssl.defaultKeyStore.expected.CWPKI0817W.useraction=Review the configuration and either add the missing keyStore, or change the default SSL configuration to use a different keystore.
+#--------------------------------------------------------------------------------------------------
+ssl.defaultSSLConfig.noSuchKeyStore.CWPKI0818E=CWPKI0818E: The default SSL configuration references a <keyStore> element with an ID of {0} which does not exist in the configuration or whose definition is missing a password. Either define a keyStore with id {0} or update the <ssl id="defaultSSLConfig" keyStoreRef="{0}" /> to reference an existing keystore.
+ssl.defaultSSLConfig.noSuchKeyStore.CWPKI0818E.explanation=The default SSL configuration expects a keyStore element which does not exist.
```

Comment:
recommend consistent keystore reference, as mentioned in the previous comment and add comma:
. . . keystore element, which does not exist.

Reply:
One last recommendation: ID and id are in the strings, let's pick one? (generally ID unless in code)